### PR TITLE
[MINOR][DOCS] Fix links to Cloud Storage connectors docs

### PR DESCRIPTION
**Pull request by medb**

### CODE REVIEW

File: docs/cloud-integration.md [hunk 1]
```diff
@@ -254,8 +254,8 @@ Here is the documentation on the standard connectors both from Apache and the cl
 * [Azure Data Lake Gen 1](https://hadoop.apache.org/docs/current/hadoop-azure-datalake/index.html).
 * [Hadoop-AWS module (Hadoop 3.x)](https://hadoop.apache.org/docs/current3/hadoop-aws/tools/hadoop-aws/index.html).
 * [Amazon S3 via S3A and S3N (Hadoop 2.x)](https://hadoop.apache.org/docs/current/hadoop-aws/tools/hadoop-aws/index.html).
```
Comment (kiszk) [inline docs/cloud-integration.md:256]: Could you please update this line, too? This is because s3n connector has been removed.

Comment (medb) [reply docs/cloud-integration.md]: I have updated link to reference Hadoop 2, which still supports S3N even though it's deprecated. Let me know if it's better to remove reference to S3N.

Comment (kiszk) [reply docs/cloud-integration.md]: Thank you for updating the link. It is fine with me.

**Comment by srowen** [comment]
Jenkins test this please

**Comment by srowen** [comment]
Looking good @medb. I updated the title per https://spark.apache.org/contributing.html but it's a minor thing. We usually ask for a bit of a description but I get this is a pretty straightforward small change.

**Review by dongjoon-hyun — APPROVED** [review]
+1, LGTM, too.
Thank you for your first contribution, @medb .
Thank you, @kiszk and @srowen . Merged to master/3.0.

**Pull request closed without merging**